src/peergos/server/tests/MultiNodeNetworkTests.java

```diff
@@ -92,7 +94,7 @@ public static void init() throws Exception {
                     .with("social-sql-file", ":memory:");
             Main.PEERGOS.main(normalNode);
 
-
+            Thread.sleep(10_000);
```

Comment:
why the sleep() call

Reply:
Only way I could get the ipfs swarm connect calls not to error all the time.

Reply:
(the ipfs daemon, despite being able to answer id calls over http, can't correctly answer this call yet)